assets/js/modules/tagmanager/components/settings/SettingsForm.js

```diff
@@ -45,6 +76,16 @@ export default function SettingsForm() {
 				<WebContainerSelect />
 
 				<AMPContainerSelect />
+
+				{ isResolvingWebGetLiveContainerVersion ||
+				isResolvingAMPGetLiveContainerVersion ? (
+					<div className="googlesitekit-margin-left-1rem googlesitekit-align-self-center">
```

Comment:
I'm guessing these classes were copied from the reference element in `CompatibilityChecks` but they don't seem to be necessary here due to the placement in the grid.

![image](https://user-images.githubusercontent.com/1621608/148380763-43b1cbfe-af5d-485b-9cb2-1447ab2db0ba.png)